skrifa/src/outline/glyf/mod.rs

```diff
@@ -1159,10 +1171,16 @@ impl Scaler for HarfBuzzScaler<'_> {
                 .get_mut(delta_base..delta_base + count)
                 .ok_or(InsufficientMemory)?;
             if deltas::composite_glyph(gvar, glyph_id, self.coords, &mut deltas[..]).is_ok() {
-                // If the font is missing variation data for LSBs in HVAR then we
-                // apply the delta to the first phantom point.
-                if !self.outlines.has_var_lsb {
-                    self.phantom[0].x += deltas[count - 4].x;
+                // Apply selective deltas to phantom points.
+                match self.outlines.var_metrics {
+                    AvailableVarMetrics::Missing => {
+                        self.phantom[0].x += deltas[count - 4].x;
+                        self.phantom[1].x += deltas[count - 3].x;
+                    }
+                    AvailableVarMetrics::MissingSideBearings => {
+                        self.phantom[0].x += deltas[count - 4].x;
+                    }
```

Comment:
Reading deltas at `count - magic #` is recurring, I wonder if there is an opportunity to make a helper fn or newtype with a nicely named (and probably always inlined) accessor?